server/http.go

```diff
@@ -283,12 +329,17 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 		return
 	}
 
+	err := p.setUserStatus(post.UserId, int(meetingID), true)
+	if err != nil {
+		p.API.LogDebug("failed to change user status", "error", err)
```

Comment:
Can we return the error, and handle it on the caller's side? I think we will need to make a separate small function to do so.